.github/workflows/main.yml

```diff
@@ -205,15 +205,18 @@ jobs:
     - build-wrappers-linux
     - build-wrappers-windows-uwp
     outputs:
-      nuget_pkg_version: ${{ steps.find-nupkg-version.outpus.NUGET_PKG_VERSION }}
-      build_suffix: ${{ github.event_name == 'pull_request' && format('PR-{0}.{1}', github.event.number, env.GITHUB_RUN_NUMBER) || format('alpha.{0}', env.GITHUB_RUN_NUMBER)}}
+      nuget_pkg_version: ${{ steps.find-nupkg-version.outpus.nuget_pkg_version }}
```

Comment:
```suggestion
      nuget_pkg_version: ${{ steps.find-nupkg-version.outputs.nuget_pkg_version }}
```

Reply:
🤦‍♂️

Reply:
That's why we should have strongly typed languages everywhere 😄

Reply:
Strongly agree and also stronger glasses for me 😅.
I fixed this, let me know if there is any more feedback.